src/views/SelectFacility.vue

```diff
@@ -0,0 +1,225 @@
+          </ion-button>
+          <ion-button fill="clear" class="mobile-only">
+            <ion-icon :icon="filterOutline" />
+          </ion-button>
+          <ion-button class="desktop-only" @click="() => router.push('/select-facility-csv-upload')">{{ $t("Upload CSV") }}</ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div class="find">
+        <section class="search">
+          <ion-searchbar :placeholder="$t('Search orders')" />
+        </section>
+
+        <aside class="filters desktop-only">
+          <ion-list>
+            <ion-list-header>{{ $t("Filter stores") }}</ion-list-header>
+            <ion-item>
+              <ion-label>{{ $t("State") }}</ion-label>
+              <ion-select value="all">
+                <ion-select-option value="all">All</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item>
+              <ion-label>{{ $t("Type") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">Any</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </ion-list>
+
+          <ion-list>
+            <ion-list-header>{{ $t("Selected products") }}</ion-list-header>
+            <ion-item-divider>
+              Parent
+              <ion-checkbox slot="end"/>
+            </ion-item-divider>
+            <ion-item>
+              <ion-label>Variant name</ion-label>
+              <ion-checkbox slot="end" />
+            </ion-item>
+            <ion-item>
+              <ion-label>Variant name</ion-label>
+              <ion-checkbox slot="end" />
+            </ion-item>
+            <ion-item>
+              <ion-label>Variant name</ion-label>
+              <ion-checkbox slot="end" />
+            </ion-item>
+          </ion-list>
+        </aside>
+
+        <main class="main">
+          <section class="sort"></section>
+
+          <section class="section-header"></section>
+
+          <ion-item lines="none">
+            <ion-label>{{ $t("Stores") }}</ion-label>
+          </ion-item>
+
+          <section class="section-grid">
+            <ion-card>
+              <ion-item-divider>
+                Store Group
+                <ion-checkbox slot="end"/>
+              </ion-item-divider>
+              <ion-item>
+                <ion-label>Store name</ion-label>
+                <ion-checkbox slot="end" />
+              </ion-item>
+              <ion-item>
+                <ion-label>Store name</ion-label>
+                <ion-checkbox slot="end" />
+              </ion-item>
+              <ion-item>
+                <ion-label>Store name</ion-label>
+                <ion-checkbox slot="end" />
+              </ion-item>
+            </ion-card>
+            <ion-card>
+              <ion-item-divider>
+                Store Group
+                <ion-checkbox slot="end"/>
```

Comment:
@azkyakhan When improving code it's important to have a wholistic approach. You shouldn't just edit the exact comment I made. Instead you have to be proactive about the improvements. 

If you remove slot end selector from css you should naturally remove it from markup too.

Reply:
If I am removing `end` slot from markup, Its looking like this - 
![checkbox-issue](https://user-images.githubusercontent.com/82817616/151693192-b9e5094b-cf92-470f-901d-5052881f8dbd.png)

that is why I havenot removed it from markup after removing from selector.

Reply:
Thanks for the explanation. Next time you can instead reference documentation and mention support for slots in ion-item dividers. That is more helpful: https://ionicframework.com/docs/api/item-divider#slots